app/views/devise/registrations/_external_openid_connect.html.erb

```diff
@@ -0,0 +1,21 @@
+<% if id.nil? || id.value == '' %>
+<%# If we dont have an id we need to link %>
+<i class="fas fa-user" title="<%= _('Institutional credentials') %>" aria-hidden="true"></i>
+        &nbsp;
+        <%= link_to _('Link your institutional credentials'),
+                    Rails.application.routes.url_helpers.send("user_openid_connect_omniauth_authorize_path"),
+                    title: _("Link your institutional credentials to access your account with them."),
+                    'data-toggle': "tooltip", method: :post %>
+<% else %>
+<%# If We do have and id we need to present the option to unlink %>
+<% unlinktext = _("Unlink your account from #{scheme.description}. You can link again at any time.") %>
+<% unlinkconf = _("Are you sure you want to unlink #{scheme.description} ID?") %>
+<%= id.value %>
```

Comment:
This will be more usable for users. However, at this time I would like to take more time to consider parsing the attributes, since we cannot  guaranteed that the email will be there. We will follow up this work on #887